src/swsssdk/scripts/sonic-db-cli

```diff
@@ -36,3 +57,29 @@ else:
             print "\n".join(resp)
         else:
             print resp
+
+def main():
+    parser = argparse.ArgumentParser(description='SONiC DB CLI',
+                         formatter_class=argparse.RawTextHelpFormatter)
+    parser.add_argument('db_or_op', nargs='?', type=str, help='Database name Or Unary operation')
```

Comment:
>nargs='?' [](start = 36, length = 9)

nargs=1, or should we remove nargs here?

Reply:
we can remove it though '?' is also fine